package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSON is not sorted](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=5444564)